src/buffer.c

```diff
@@ -108,8 +108,7 @@ bool zseek_buffer_resize(zseek_buffer_t *buffer, size_t size)
     if (!zseek_buffer_reserve(buffer, size))
         return false;
 
-    if (size > buffer->size)
-        memset((uint8_t*)buffer->data + buffer->size, 0, size - buffer->size);
+    // TODO OPT: zero-init new memory, if new size > old size, in bebug builds?
```

Comment:
Minor typo

Reply:
My eyes are playing games, I don't see it. Could you point it out?

Reply:
"bebug"

Reply:
Oh, indeed. Thank you, fixed!